libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2022,6 +2022,12 @@ AC_AttitudeControl = {}
 ---@return number -- yaw slew rate
 function AC_AttitudeControl:get_rpy_srate() end
 
+-- return current desired rates on roll/pitch/yaw
+---@return number -- desired roll rate
+---@return number -- desired pitch rate
+---@return number -- desired yaw rate
+function AC_AttitudeControl:get_rpy_rate() end
```

Comment:
I know the comment says it but it would be nice to include "desired" in the function name. There is only one character difference between this and the one above. Maybe just `get_desired_rate`? Dropping the rpy, although we have rpy in the slew rate one I guess.

Reply:
Possibly is should be "target" not "desired" there is a difference but I can never remember which way round it is.